### PR TITLE
created global class for sections settings, some minor code cleaning

### PR DESCRIPTION
**Pull request by JarekSopinski**
DETAILS:

* Added .section class - main settings for every section in body, excluding the one with .parallax background (sing-up). I added this class to #icons, #description, #about-us and #contact and deleted coresponding old code. I only left different padding in #contact (20px), which overwrites padding definied in .section class (50px).

* Removed usless polish comments in html (ex. 'opis funkcjonalności rozbudowany')

* Removed comment defining resolutions in css (was usless, we're using only 576 px breakpoint)

* Simplified names:

- #product-icons-section changed to #icons (updated hrefs in menu);
- .icons-display-container to .icons-container; .about-us-container to .about-us-container (previously I used word 'display' bacuse of changes in display mode, but it's actually useless and can be misleading)

* Cleaning of headers in #about-us. Removed usless properties definied by header classes in global styling and deleted class .about-us-text (not needed any longer).

* Removed text class in #icons (not needed any longer, style is definied in global styling).

### CODE REVIEW

**Review by KamilaKisiel — DISMISSED** [review]

File: index.html [hunk 1]
```diff
@@ -53,34 +53,32 @@ <h3 class="shadow">Startujemy już wkrótce!</h3>
...
 <main>
-    <section id="product-icons-section" class="flex-container-center">
-        <!-- krótki opis funkcjonalności produktu -->
+    <section id="icons" class="flex-container-center section">
         <div class="flex-container-center">
-            <div class="icons-display-container">
+            <div class="icons-container">
                 <img class="icons-images" src="images/icons/car-mechanic.svg" alt="car icon">
-                <h2 class="icons-text">Tysiące części<br>dla każdego<br>samochodu</h2>
+                <h2 >Tysiące części<br>dla każdego<br>samochodu</h2>
```
Comment (RadekRo) [inline index.html]: This one should be corrected. Do not leave empty spaces inside markup names.

**Review by RadekRo — CHANGES_REQUESTED** [review]
Please correct a little mistake inside index.html code. Check for my comment in certain code-line.

**Comment by JarekSopinski** [comment]
@RadekRo , mistake in h2 is fixed.

File: index.html (hunk 1 above)
Comment (JarekSopinski) [reply index.html]: Fixed.

**Review by RadekRo — APPROVED** [review]

**Pull request merged**